### ISSUE TITLE
`aws_ses_domain_identity` is stuck in "Still creating" state when attempting to create in an unsupported region

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-aws#732]
_This issue was originally opened by @joshuaspence as hashicorp/terraform#14141. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

In Terraform 0.9.4, attempting to create an `aws_ses_domain_identity` in an unsupported region seems to hang indefinitely. For example, according to the [documentation](http://docs.aws.amazon.com/ses/latest/DeveloperGuide/regions.html), SES isn't currently supported in the `ap-southeast-2` region.

This can be reproduced with the following Terraform configuration:

```
provider "aws" {
  region = "ap-southeast-2"
}

resource "aws_ses_domain_identity" "main" {
  domain = "example.com"
}
```

I had Terraform running for over 1 hour before I eventually realized the error of my ways.

**Comment by leventyalcin** [comment]
Hi all, 

I think this is still an issue. I was trying to do the same thing with Terraform v0.10.2 and this is the result

```
module.route53_setup.aws_ses_domain_identity.dev: Still creating... (38m40s elapsed)
```

**Comment by lewislambert** [comment]
I can confirm this is still the case in v0.10.7. Killed off terraform after about 50 minutes. Interestingly, the API call doesn't seem to show as even taking place as far as AWS is concerned.

**Comment by joernheissler** [comment]
See the duplicate #2427 for more info on this. DNS query fails with NXDOMAIN, in my opinion it should be an instant error.

**Comment by bflad** [comment]
The fix to lower the threshold for DNS resolution errors (#4459) was 
This was previously released in [version 1.18.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1180-may-10-2018) and has been available in all releases since. Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
